packages/core/src/context/MemoryStore.test.ts

```
@@ -0,0 +1,1075 @@
+        content: longMemory,
+        model: 'text-embedding-004'
+      });
+    });
+
+    it('should handle malformed embedding arrays', async () => {
+      const malformedEmbeddings = [
+        null,
+        undefined,
+        [],
+        [NaN, 0.1, 0.2],
+        [Infinity, 0.1, 0.2],
```

Biome (1.9.4)

[error] Use Number.NaN instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NaN instead.


(lint/style/useNumberNamespace)